### PR TITLE
feat(avatar): UC app integration icon added

### PR DESCRIPTION
**Pull request by 321gillian**
✅ Closes: COMUI-2664

### CODE REVIEW

File: packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss [hunk 1]
```diff
@@ -217,6 +217,16 @@
   }
 }
 
+.gux-avatar-integration-badge {
+  position: absolute;
+  inset-block-end: 0;
+  inset-inline-end: 0;
+  z-index: 2;
```
Comment (321gillian) [inline packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss:224]: Needed to bring the logo above the overlay in the gux-avatar-change-photo version. Had a look at the z-index tokens but the meaning seemed a little different so wasn't really sure which suited this purpose.

File: packages/genesys-spark-components/src/components/beta/gux-avatar/example.html [hunk 2]
```diff
@@ -108,6 +108,16 @@ <h2>Dynamic Example</h2>
...
+    <gux-form-field-dropdown>
+      <gux-dropdown value="" id="update-uc-integration">
+        <gux-listbox aria-label="Update Uc Inegration">
+          <gux-option value="zoom">Zoom</gux-option>
+          <gux-option value="team">Teams</gux-option>
+          <gux-option value="8x8">8x8</gux-option>
+        </gux-listbox>
+      </gux-dropdown>
+      <label slot="label">UcIntegration</label>
+    </gux-form-field-dropdown>
```
Comment (conor-darcy) [inline packages/genesys-spark-components/src/components/beta/gux-avatar/example.html:120]: The Teams option is not working.
Should there be a Slack option here?

File: packages/genesys-spark-components/src/components/beta/gux-avatar/example.html [hunk 3]
```diff
@@ -479,6 +480,41 @@ <h2>Notifications</h2>
...
+  ring
+  uc-integration="8x8"
+>
+</gux-avatar-beta>
+<gux-avatar-beta
+  name="Stephen Byrne"
+  presence="available"
+  label="Available"
+  ring
+  uc-integration="slack"
```
Comment (conor-darcy) [inline packages/genesys-spark-components/src/components/beta/gux-avatar/example.html:515]: Slack is not displaying anything.

File: packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss (hunk 1 above)
Comment (conor-darcy) [reply packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss:224]: The semantic tokens are not useful here plus you have created a new stacking context so no semantic token would make sense. You could use --gse-core-zIndex-1 for a value of '1' (I'm not sure why you have '2').

Anyway...it doesn't seem to do anything, is it needed at all?

File: packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx [hunk 4]
```diff
@@ -4,8 +4,10 @@ import { logWarn } from '@utils/error/log-error';
 import {
   GuxAvatarSize,
   GuxAvatarAccent,
-  GuxAvatarPresence
+  GuxAvatarPresence,
+  GuxAvatarUcIntegrationApps
 } from './gux-avatar.types';
+import { render8x8SVG, renderTeamsSVG, renderZoomSVG } from './svg-utils';
```
Comment (conor-darcy) [inline packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx:10]: Should we have Slack too?

File: packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx [hunk 5]
```diff
@@ -157,6 +165,29 @@ export class GuxAvatar {
...
+
+  private renderUcIntegrationBadge(): JSX.Element | null {
+    if (this.ucIntegration && this.size === 'large') {
+      return (
+        <div class="gux-avatar-integration-badge">
+          {this.renderUcIntegrationsIcon()}
+        </div>
+      ) as JSX.Element;
+    }
+  }
```
Comment (conor-darcy) [inline packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx:189]: Should we throw a warning if ucIntegration is true but size is not large?

File: packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss (hunk 1 above)
Comment (321gillian) [reply packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss:224]: It's basically to account for the situation where it's used in combination with gux-avatar-change-photo. In that case, the overlay has a z-index of 1 to pop it above the avatar. Then the app logo will be partially hidden if it has the same z-index as the rest of the avatar. 

`  <gux-avatar-focusable-beta>
    <gux-avatar-change-photo-beta>
      <gux-avatar-beta
        slot="avatar"
        name="Conor Darcy"
        presence="available"
        badge
        ring
      ></gux-avatar-beta>
    </gux-avatar-change-photo-beta>
  </gux-avatar-focusable-beta>`

File: packages/genesys-spark-components/src/components/beta/gux-avatar/example.html (hunk 3 above)
Comment (321gillian) [reply packages/genesys-spark-components/src/components/beta/gux-avatar/example.html]: Sorry, I just put slack in as an example of an invalid app. I have changed it to be clearer. There is no Slack logo requirement.

File: packages/genesys-spark-components/src/components/beta/gux-avatar/example.html (hunk 2 above)
Comment (321gillian) [reply packages/genesys-spark-components/src/components/beta/gux-avatar/example.html:120]: Fixed the teams option

File: packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx (hunk 5 above)
Comment (321gillian) [reply packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx:196]: Added

File: packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss (hunk 1 above)
Comment (conor-darcy) [reply packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss:224]: Ok, gotcha. Maybe you could use the same token you have for the badge? I'm sure you can have multiple elements on the same level.

**Review by conor-darcy — APPROVED** [review]

File: packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss (hunk 1 above)
Comment (321gillian) [reply packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss:224]: Of course, should've copped that. Updated now

**Pull request merged**